lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -63,6 +64,10 @@
         valid=specs.is_in("urllib", "curl", None),
         default=None
     ),
+    transport_timeout=dict(
+        map=lambda val: None if val == "None" else int(val),
```

Comment:
I'm fine changing this, but it's used in other pulsar runner params as well and I probably shouldn't change these for a stable PR?